### PR TITLE
Cosmetic burner

### PR DESCRIPTION
**Pull request by pixar02**
Added a Bunsen burner
Right click to toggle blue flame (adds/removes oxygen)
Flint and steel to turn on
Water should turn of the flame
Same behavior as Microscope and tuberack (height changes on lab counters)

### CODE REVIEW

**Review by JoeyDP — APPROVED** [review]
Code looks good. Haven't tested in game.

**Review by smtdm — CHANGES_REQUESTED** [review]
Cannot turn off the bunsenburner? Right-clicking with a bucket of water just places the water against the bunsenburner.
Add fire charge to the list of items that can light the bunsenburner

File: src/main/java/be/uantwerpen/minelabs/block/LabBlock.java [hunk 1]
```diff
@@ -63,6 +65,7 @@ public BlockState mirror(BlockState state, BlockMirror mirror) {
 
     @Override
     public BlockState getPlacementState(ItemPlacementContext ctx) {
+        // TODO TRIGGER NEIGHBOR UPDATE
```
Comment (JoeyDP) [inline src/main/java/be/uantwerpen/minelabs/block/LabBlock.java:68]: I think neighbour updates are triggered by MC, you don't have to manually call it.

**Review by JoeyDP — APPROVED** [review]

**Review by JoeyDP — APPROVED** [review]

**Review by smtdm — APPROVED** [review]
Works as intende

**Pull request merged**